### DISCUSSION TITLE
IP address

### DISCUSSION THREAD

**Discussion by bingobob** [vicwomg/pikaraoke#255] [Q&A]
Hi,

My pi has a 192.168 address and an IP that you can get to from the internet.

How do I override the displayed IP and QR code and display another.. I'd like to show the public IP on the screen basically and not the 192 one.

The QR code should go there also

?

**Comment by jramboz** [comment]
I actually started to code up a patch to do this a while back. Let me see if I can brush it off and put in a PR.
